configuration.yaml

```diff
@@ -0,0 +1,65 @@
+global:
+  logDir: gs://ot-team/irene/evidence_test
```

Comment:
We should remove the explicit destination of the log files. The logfile can either be a file in the working directory or specified via a command line option.

Reply:
New location for the logs: `otar000-evidence_input/parser_logs`